worldedit-core/src/main/java/com/sk89q/worldedit/extent/world/SurvivalModeExtent.java

```diff
@@ -96,7 +96,9 @@ public void setStripNbt(boolean stripNbt) {
         } else {
             // Can't be an inlined check due to inconsistent generic return type
             if (stripNbt) {
+                //FAWE start - Use CompoundBinaryTag
                 return super.setBlock(location, block.toBaseBlock((CompoundBinaryTag) null));
+                //FAWE end
```

Comment:
just remove the cast?

Reply:
We use Adventure NBT in much more spots upstream does (until now). Once they move from CompoundTag to CBT on named method, we can remove the annotations.
Currently we delegate BlockStateHolder#toBaseBlock to LazyReference<CompoundBinaryTag> so removing the cast here wont work out well.

Reply:
it's casting null though